src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTab }) => {
+  function onClickHandle(tab) {
```

Comment:
do not use `function` keyword in react components. use ES6 syntax instead (it gives more opportunities)
```suggestion
  const handleTabChange = (tab) => {
```